pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -449,16 +451,40 @@ func (r *PerformanceProfileReconciler) Reconcile(ctx context.Context, req ctrl.R
 	})
 	err = r.Client.Get(context.Background(), key, nodeCfg)
 	if err != nil {
-		klog.Errorf("failed to get config node object; name=%q err=%v", nodeCfg.GetName(), err)
-		nodeCfg.Name = nodeCfgName
-		nodeCfg.Spec.CgroupMode = apiconfigv1.CgroupModeV1
-		//nolint:errcheck
-		r.Client.Update(ctx, nodeCfg)
+		return reconcile.Result{}, err
 	}
+
 	if nodeCfg.Spec.CgroupMode != apiconfigv1.CgroupModeV1 {
+		klog.Infof("Switching cluster to cgroupv1")
```

Comment:
4.15 wise,  we might need to perform the switch here back to cgroupv2 (after this PR is merged and cherry picked)

Reply:
Is 4.15 targeting cgroupv2 for NTO?